addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -314,6 +314,13 @@
 
         <div id="o_wslides_sidebar_collapse" class="collapse d-md-block">
             <table class="table table-sm mt-3">
+                <tr>
+                    <th class="border-top-0">Responsible</th>
+                    <td class="border-top-0">
+                        <a t-if="channel.responsible_email" t-attf-href="mailto:#{channel.responsible_email}" t-field="channel.user_id"/>
```

Comment:
I would not add `t-field="channel.user_id"` on the email link, that might lead to an unexpected behavior.